src/util/index.js

```diff
@@ -21,5 +21,18 @@ import axios from "axios";
     }
   };
 
+  const [message, setMessage] = useState('');
```

Comment:
None of this code is useful (the message state or the useEffect below) and therefore I think we should remove it. State is only useful inside of a component, which it is not here.